x-pack/legacy/plugins/epm/server/packages/index.ts

```diff
@@ -18,6 +18,7 @@ export {
   getCategories,
   getFile,
   getInstallationObject,
+  getInstallation,
```

Comment:
This will disappear as soon as https://github.com/elastic/kibana/pull/52628 is merged.